tool-plugins/nextjs-starter/src/pages/401.tsx

```diff
@@ -9,5 +9,5 @@ export default function Error401() {
 		}
 	}, []);
 
-	return <span>Error: Unauthorized</span>;
+	return <div></div>;
```

Comment:
is this on purpose?

Reply:
Actually yes. In a normal situation, we see that error message for a split second, which gives a bad impression. There could be a situation that we really need to display 401 error but we haven't got that case yet. So I think this makes more sense.